CHANGES.rst

```diff
@@ -157,11 +157,11 @@ New Features
 
 - ``astropy.wcs``
 
-  - The included version of wcslib has been upgraded to 5.6.
+  - The included version of wcslib has been upgraded to 5.8.pre.
 
     The minimum required version of wcslib in the 4.x series remains 4.24.
 
-    The minimum required version of wcslib in the 5.x series is 5.6.
+    The minimum required version of wcslib in the 5.x series is 5.8.pre.
```

Comment:
Perhaps also add a note here (or maybe in the backwards incompatibility section) indicating the ImportError behavior?  Just so people know that it's on purpose new behavior as opposed to them doing something wrong with their wcslib install or something.